frontend/src/features/permits/pages/Amend/AmendPermit.tsx

```diff
@@ -131,7 +131,7 @@ export const AmendPermit = () => {
   };
   const fullSearchRoute = `${searchRoute}${getBasePermitNumber()}`;
 
-  const goHome = () => navigate(searchRoute);
+  const goHome = () => navigate(-1);
```

Comment:
Why this?

Reply:
searchRoute is missing the search value and search entity parameters.